clients/include/testing_matmul.hpp

```diff
@@ -1929,13 +1929,19 @@ void testing_matmul_with_bias(const Arguments& arg,
         if(arg.scaleA == hipblaslt_scaling_format::Scalar
            || arg.scaleA == hipblaslt_scaling_format::Vector)
         {
-            hipblaslt_init(hScaleA[i].buf(), size_scaleAVec[i], 1, size_scaleAVec[i], Talpha);
+            if(!arg.unit_check)
```

Comment:
why didn't just use "if(arg.norm_check)    hipblaslt_init_small() "?

Reply:
I am indifferent, the only one of concern is unit check. In rocBLAS, we have other valid options such as res_check. I'll change it.